sdk/storage/Azure.Storage.DataMovement/tests/TransferManagerTests.cs

```diff
@@ -11,13 +11,48 @@
 using System.Threading.Tasks;
 using Azure.Core;
 using Azure.Core.Pipeline;
+using Azure.Storage.DataMovement.Tests.Shared;
 using Moq;
 using NUnit.Framework;
 
 namespace Azure.Storage.DataMovement.Tests;
 
 public class TransferManagerTests
 {
+    public enum TransferDirection
```

Comment:
We already have an enum for this:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.DataMovement/tests/Shared/TransferDirection.cs